packages/cozy-search/src/components/Search/useFetchResult.jsx

```diff
@@ -65,7 +69,17 @@ export const useFetchResult = searchValue => {
           primary: r.title,
           secondary: r.subTitle,
           onClick: () => {
-            window.open(r.url)
+            if (r.slug === client.appMetadata.slug) {
+              try {
+                const url = new URL(r.url)
+                const hash = url.hash.replace('#', '')
```

Comment:
maybe we can use something from react-router-dom to deal with the hash and so avoid to remove it 🤔

Reply:
I tried different way, and react-router only understand react-router. In v6 or v7 nothing exists for example to do : `navigate("https://example.com/#/mypage")`. react-router does not understand  that we want to do `navigate("/mypage")`